crates/networking/p2p/src/config.rs

```diff
@@ -0,0 +1,14 @@
+use std::net::IpAddr;
+
+use ream_discv5::config::DiscoveryConfig;
+use ream_gossipsub::config::GossipsubConfig;
+
+pub struct NetworkConfig {
+    pub socket_address: IpAddr,
+
+    pub socket_port: u16,
+
+    pub disc_config: DiscoveryConfig,
```

Comment:
```suggestion
    pub discv5_config: Discv5Config,
```

do this in other places as well